node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -181,33 +180,35 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val nodeDiscoveryEffect: NodeDiscovery[Task] =
     new KademliaNodeDiscovery[Task](src, defaultTimeout)
 
+  if (Files.notExists(casperPersistencePath)) Files.createDirectories(casperPersistencePath)
+  val config =
+    coop.rchain.blockstorage.LMDBBlockStore.Config(casperPersistencePath, 1024 * 1024 * 1024)
```

Comment:
The store size should probably also be configurable. I chose to construct the Casper config object from the node config object in [`conf.scala`](https://github.com/rchain/rchain/blob/dev/node/src/main/scala/coop/rchain/node/conf.scala#L247). You could consider doing the same.